Earth Data Hub notebooks for desp-lab on Insula

nicolamasotti opened the pull request:
This is a couple of test notebooks to be added to Insula desp-lab. These notebooks serve as simple tutorials to better understand the usage of EDH datasets and leverage the power of Zarr based workflows.

In order to run the notebooks, one must retrieve its Earth Data Hub personal access token from [Earth Data Hub account settings ](https://earthdatahub.destine.eu/account-settings) and place it in the notebooks under the `PAT` variable. This is further described in each notebook introduction.

gcipolletta commented:
Dear @nicolamasotti 

Thank you so much for this contribution.

We tested it on Insula Code Lab environment and report here just a couple of notes. Please consider to implement these and open a second pull-request for fixing.

1) On `tutorial-reanalysis-era5-land-floods-precipitation-anomaly.ipynb` the cell #2 gathering data does not report the correct URL, causing an HTTP 401 failure. Please fix it to be as in the other notebook you provided.
2) In both notebooks, in the introduction markdown preceding the first cell, please specify to place the token in quotation marks (“ ”); otherwise, the user might encounter a syntax error.

Also, we would like to add this contribution within the Section "Credits" in the general repository readme (https://github.com/SercoSPA/DESP-UserWorkflowService-Templates?tab=readme-ov-file#credits). If you have any specific suggestion for EDH credits or if you want to edit on your own please let us know.

Kind regards
Gaia

nicolamasotti commented:
Everything should be fixed in [this](https://github.com/SercoSPA/DESP-UserWorkflowService-Templates/pull/29) pull request 

I have also renamed the notebooks and updated credits (but feel free to change them if necessary)
